_component_design/vertical-stepper.md

```diff
@@ -4,31 +4,257 @@ parent: Content Areas
 
-## Step 1
+## Introduction
 
-{% figure [caption:""] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/stepper-1.svg){:width="100%"}
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+
+- The stepper pattern is used for complex workflows that require multiple inputs from a user to complete. The complex workflow is segmented or chunked into thematic steps and then progressively revealed to the user via the stepper pattern. Vertical steppers fulfill the same need as a horizontal stepper; however, the orientation yields itself to work within all screen sizes.
+
+### Best practices
+
+- Ensure a stepper has at minimum 3 steps and at maximum 5 steps.
+- Name each step thematically based on the inputs contained within a step. For example, a step dealing with inputs **Username**, **First name**, **Address**, etc. could be labeled **Account Info** or **User Details**. This ensures users are primed for the type of information that will be presented in a step when first skimming a stepper form.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"The vertical stepper component"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-hero.png){: width="1440px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Overview
+
+The vertical stepper component is comprised of the following pieces:
+
+- Steps
+- Step content area (this can be comprised of any components from the inputs and controls section)
+- Step counter
+- Step dividers
+- Step revealer
+- Step selection	
+- Step title	
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+
+{% figure [caption:"Vertical stepper composition (NOT TO SCALE)"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-composition.png){: width="900px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+## Pattern specifications
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Spacing
+
+{% figure [caption:"Spacing specifications for the vertical stepper pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/vert-stepper/vert-stepper-spacing.png){: width="903px"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+### Specifications
```

Comment:
Is this spec section specific to spacing?

Reply:
I'm unclear what you mean in this comment.

Reply:
Is this section the specifications for all elements of the Vertical Stepper pattern, or is this section just in regards to specifications for Spacing in the Vertical Stepper Pattern? Just confused by the flow of information.

Reply:
I know the info-dev team is going to be doing a content audit this upcoming sprint. Can we address this concern during that process?

Reply:
Sounds good.